test/functional/feature_init.py

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env python3
-# Copyright (c) 2021-2022 The Bitcoin Core developers
+# Copyright (c) 2021-present The Bitcoin Core developers
```

Comment:
running this command 
`grep -nr --text "\-present" ./src`

I only see these files using `present`
```
./src/streams.cpp:1:// Copyright (c) 2009-present The Bitcoin Core developers
./src/kernel/mempool_removal_reason.cpp:1:// Copyright (c) 2016-present The Bitcoin Core developers
./src/kernel/mempool_removal_reason.h:1:// Copyright (c) 2016-present The Bitcoin Core developers
```

if this is something we're going towards would we want to update the copy right of all files whenever there is a change to the file in any future PR's?